core/keyboard_nav/ast_node.ts

```diff
@@ -694,10 +694,15 @@ export class ASTNode {
     return astNode;
   }
 }
-export interface Params {
-  wsCoordinate: Coordinate;
+
+export namespace ASTNode {
+  export interface Params {
+    wsCoordinate: Coordinate;
+  }
```

Comment:
Mmm I think this wasn't changed because the identifier isn't actually used as a type anywhere. E.g. in this file we have:

```
    /**
     * The type of the location.
     * One of ASTNode.types
     * @type {string}
     * @private
     */
    this.type_ = type;
```

I think this is a bigger problem, and should probably be solved in a follow-up PR, because it requires more work to fix all of the types.